integration/test-bundle.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Some doc and improvements would be good, I have no idea what this is doing and what is the expected output of this.